index.html

```diff
@@ -166,6 +166,21 @@
 If certain groups are deliberately excluded from using these features, explain
 why.
 
+<aside class="example" title="Excluded Users" id="example-exclusion">
+
+Example: The [Web GPU specification](https://www.w3.org/TR/webgpu/)
```

Comment:
This block will already start with "EXAMPLE:", so we don't need it again in the body:

```suggestion
The [Web GPU specification](https://www.w3.org/TR/webgpu/)
```

Reply:
There are other blocks in here that explicitly say `Example: ....` should we change those too?

Reply:
It looks redundant in https://w3ctag.github.io/societal-impact-questionnaire/#example-measuring and https://w3ctag.github.io/societal-impact-questionnaire/#example-surveillance, so I'd remove it from there too.